tsl/src/nodes/vector_agg/exec.h

```diff
@@ -27,7 +27,9 @@ typedef struct GroupingColumn
 {
 	int input_offset;
 	int output_offset;
-	int value_bytes;
+
+	int16 value_bytes;
```

Comment:
Nit: this is only in-memory, and it would require a sign-extension each time you set the value, so there is no gain from using `int16` rather than just `int`. Might even end up with being slower.

Reply:
yeah, it was not intentional to save anything. I think I just removed and added back later and used the type from pg_attribute.attlen

Reply:
Actually, this is from the extra commit by @akuzm that I was asked to include in this PR, so I'll keep as is.